pkg/tkctl/debug/launcher.go

```diff
@@ -140,6 +140,12 @@ func (l *Launcher) createContainer(command []string) (*container.ContainerCreate
 		StdinOnce:  true,
 	}
 	hostConfig := &container.HostConfig{
+		// share host debugfs, kernel modules and kernel sources path to container for kernel debugging
+		Binds: []string{
+			"/sys/kernel/debug:/sys/kernel/debug:rw",
```

Comment:
This directory is not always available. I think we need an option in the command line.

Reply:
Good point. But IIUC,  it doesn't hurt?  Non-existent bind mounts from the docker engine will get initialized to an empty directory owned by root. Adding an option to the command line doesn't guarantee the existence. Users have to check the existence before it specify the option?  So leave it will be better?